content/docs/deployment/_index.md

```diff
@@ -1,13 +1,64 @@
 ---
 title: "Deployment"
 linkTitle: "Deployment"
+no_list: true
 description: Deployment of CSM for Replication
 weight: 1
 ---
+The Container Storage Modules along with the required CSI Drivers can each be deployed using CSM operator.
 
+>Note: Currently CSM operator is in tech preview and is not supported in production environments.
+
+{{< cardpane >}}
+  {{< card header="[**CSM Operator**](csmoperator/)"
```

Comment:
@shanmydell, Please help modify CSM Operator to Container Storage Modules Operator in the card header. I was about to make this change and then saw this PR where the deployment page itself is revamped. 
Thanks in advance